Userland/Libraries/LibJS/Runtime/NativeFunction.h

```diff
@@ -1,12 +1,15 @@
 /*
  * Copyright (c) 2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2021-2022, Linus Groh <linusg@serenityos.org>
```

Comment:
Should maybe just be `2022`? You probably touched the file last year so I don't think it really matters

Reply:
Yeah, I worked on it a bit last year, so just 2022 seems odd. :^)

Reply:
Don't want `2022` to get lonely